lib/ui/natives.dart

```diff
@@ -36,15 +36,30 @@ void _setupHooks() {
   }());
 }
 
-void saveCompilationTrace(String filePath) {
+/// Returns runtime Dart compilation trace as a memory buffer.
+///
+/// The buffer contains a list of symbols compiled by the Dart JIT at runtime up to the point
+/// when this function was called. This list can be saved to a text file and passed to tools
+/// such as `flutter build` or Dart `gen_snapshot` in order to precompile this code offline.
+///
+/// The list has one symbol per line of the following format: `<namespace>,<class>,<symbol>\n`.
```

Comment:
one symbol per line implies it's text, but this returns bytes. What's the character encoding? UTF-8?

Reply:
It's text, and I presume it's UTF-8 since it's fetched from Dart VM.